core/storage/src/lib.rs

```diff
@@ -20,6 +20,7 @@ use protocol::traits::{
 use protocol::types::{Block, Hash, Proof, Receipt, SignedTransaction};
 use protocol::Bytes;
 use protocol::{ProtocolError, ProtocolErrorKind, ProtocolResult};
+use std::convert::From;
```

Comment:
sorts `use`

Reply:
The imports from std should be in the first block.

Reply:
sorted